kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -413,7 +408,7 @@ private void commitFiles() {
       commitFile(entry.getKey());
       log.debug("Committed {} for {}", entry.getValue(), tp);
     }
-    offsetToCommit = nextOffsetToCommit;
+    offsetToCommit = currentOffset + 1;
```

Comment:
Do you add 1 because this offset gets committed to offset storage, and upon restart that's where the connector will start reading?